ios/brave-ios/Sources/Brave/Frontend/Browser/SearchResultAdClickedInfoBar.swift

```diff
@@ -0,0 +1,171 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import DesignSystem
+import Foundation
+import Shared
+import SnapKit
+import SwiftUI
+import UIKit
+
+struct SearchResultAdClickedInfoBarUX {
+  static let toastHeight: CGFloat = 100.0
+  static let toastPadding: CGFloat = 10.0
+  static let toastCloseButtonWidth: CGFloat = 20.0
+  static let toastLabelFont = UIFont.systemFont(ofSize: 15, weight: .semibold)
+  static let toastBackgroundColor = UIColor(braveSystemName: .schemesOnPrimaryFixed)
+  static let learnMoreUrl = "https://support.brave.com/hc/en-us/articles/360026361072-Brave-Ads-FAQ"
+}
+
+class SearchResultAdClickedInfoBar: Toast, UITextViewDelegate {
+  let tabManager: TabManager
+
+  init(tabManager: TabManager) {
+    self.tabManager = tabManager
+
+    super.init(frame: .zero)
+
+    self.tapDismissalMode = .outsideTap
+
+    self.clipsToBounds = true
+    self.addSubview(
+      createView(
+        Strings.searchResultAdsClickedInfoBarTitle + " "
+      )
```

Comment:
Missing constraints on the added toast view

Reply:
Fixed